### PR TITLE
set bittorrent seed-ratio to 100 by default; warning user to mount /app/conf

### PR DESCRIPTION
**Pull request by yingshaoxo**
As you wish.

### CODE REVIEW

File: conf/aria2.conf [hunk 1]
```diff
@@ -13,8 +13,11 @@ rpc-allow-origin-all=true
 rpc-listen-all=true
 #RPC端口, 仅当默认端口被占用时修改
 #rpc-listen-port=6800
-#最大同时下载数(任务数), 路由建议值: 3
-max-concurrent-downloads=5
+#最大同时下载数(任务数), 路由建议值: 3。
+#但为了更多的进行torrent共享，为世界作贡献，我们把它设为100
+max-concurrent-downloads=100
```
Comment (wahyd4) [inline conf/aria2.conf]: Can I ask why you set to 100?

File: README.md [hunk 2]
```diff
@@ -104,6 +104,8 @@ services:
 ```
 Then just run `docker-compose up -d`, that's it!
 
+> Warning: if you don't mount `/app/conf`, whenever the container restart, you'll lose your downloading progress.
```
Comment (wahyd4) [inline README.md]: Maybe just move the content to `Supported Volumes` section? 

Besides, can you please also update the Chinese version readme as well? 

Thanks.

File: conf/aria2.conf (hunk 1 above)
Comment (yingshaoxo) [reply conf/aria2.conf]: 我有一个5T硬盘，既然要共享，那就弄多一点。5个下载太少了，如果想添加新下载，就得删旧共享，太麻烦了。

Comment (wahyd4) [reply conf/aria2.conf]: But it doesn't sound like a common requirement, so would you like to revert the change of `max-concurrent-downloads`? 

But you can always have your own setting to `100`.

**Comment by wahyd4** [comment]
@yingshaoxo Thanks a lot, can you also please update the Chinese version readme?

**Comment by yingshaoxo** [comment]
@wahyd4 
I think you are biased. The Chinese version of README did mention the problem of losing download progress. But the English version didn't.

**Comment by yingshaoxo** [comment]
> @yingshaoxo Thanks a lot, can you also please update the Chinese version readme?

There's no need for updating the Chinese version of README.

**Comment by wahyd4** [comment]
@yingshaoxo Thanks for your update. I think I just forgot to add that information before.

**Pull request merged**